doc/tasks/doctrine_orm.md

```diff
@@ -0,0 +1,32 @@
+# Doctrine ORM
+
+The Doctrine ORM task will validate that your Doctrine mapping files and check if the mapping is in sync with the database.
+It lives under the `doctrine` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        doctrine_orm:
+            skip_mapping: false
+            skip_sync: true
+            triggered_by: ['php', 'xml', 'yml']
+```
+
+**skip_mapping**
+
+*Default: false*
+
+With this parameter you can skip the mapping validation check.
+
+**skip_sync**
```

Comment:
Small question: Why is this sync skipping enabled by default? I prefer to use the defaults of the actual commandline tool.